akka-stream/src/main/scala/akka/stream/scaladsl/Graph.scala

```diff
@@ -18,8 +18,9 @@ object Merge {
    * Create a new `Merge` with the specified number of input ports.
    *
    * @param inputPorts number of input ports
+   * @param eagerClose if true, the merge will complete as soon as one of its inputs completes.
    */
-  def apply[T](inputPorts: Int): Merge[T] = new Merge(inputPorts)
+  def apply[T](inputPorts: Int, eagerClose: Boolean = false): Merge[T] = new Merge(inputPorts, eagerClose)
```

Comment:
there is a table of stages and their close semantics, you will need to update that one too.